src/transformStateWithClones.js

```diff
@@ -8,6 +8,38 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = [];
+  let clone = {};
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
Use for...in and for...of (google difference between them if needed) to get rid of index accessing
```suggestion
  for (const action in actions) {
```